### ISSUE TITLE
Causes error with unused entities

### ISSUE THREAD

**Issue by danlinn** [enriqg9/dual-thermostat#3]
Trying to show unused entities with this config resulted in a JS error and nothing shown:
```
 - entities:
      cool: climate.2gig_technologies_ci300e_thermostat_cooling_1
      heat: climate.2gig_technologies_ci300e_thermostat_heating_1
    fan_control: true
    max_slider: 80
    min_slider: 60
    name: Thermostat
    type: 'custom:dual-thermostat'```

**Comment by danlinn** [comment]
Formatting on that is off, but you get the idea :)

**Comment by enriqg9** [comment]
Fixed with [0.2.0](https://github.com/enriqg9/dual-thermostat/releases/tag/0.2.0).
